set.mm

```diff
@@ -22632,89 +22703,142 @@ of the unique existential quantifier (note that ` y ` and ` z ` need not
-      ( wnf weq wal wn adantr nfmod2 ) ABCDEABCGCDHCIJFKL $.
   $}
 
   ${
     nfeu.1 $e |- F/ x ph $.
-    $( Bound-variable hypothesis builder for uniqueness.  Note that ` x ` and
-       ` y ` needn't be distinct.  (Contributed by NM, 8-Mar-1995.)  (Revised
-       by Mario Carneiro, 7-Oct-2016.) $)
+    $( Bound-variable hypothesis builder for the unique existential quantifier.
+       Note that ` x ` and ` y ` need not be disjoint.  (Contributed by NM,
+       8-Mar-1995.)  (Revised by Mario Carneiro, 7-Oct-2016.) $)
     nfeu $p |- F/ x E! y ph $=
       ( weu wnf wtru nftru a1i nfeud mptru ) ACEBFGABCCHABFGDIJK $.
-
-    $( Bound-variable hypothesis builder for the at-most-one quantifier.
-       (Contributed by NM, 9-Mar-1995.) $)
-    nfmo $p |- F/ x E* y ph $=
-      ( wmo wnf wtru nftru a1i nfmod mptru ) ACEBFGABCCHABFGDIJK $.
   $}
 
   ${
-    $d x y $.  $d y ph $.  $d y ps $.  $d y ch $.
     eubid.1 $e |- F/ x ph $.
     eubid.2 $e |- ( ph -> ( ps <-> ch ) ) $.
-    $( Formula-building rule for unique existential quantifier (deduction
+    $( Formula-building rule for the unique existential quantifier (deduction
        form).  (Contributed by NM, 9-Jul-1994.) $)
     eubid $p |- ( ph -> ( E! x ps <-> E! x ch ) ) $=
-      ( vy weq wb wal wex weu bibi1d albid exbidv eu6 3bitr4g ) ABDGHZIZDJZGKCR
-      IZDJZGKBDLCDLATUBGASUADEABCRFMNOBDGPCDGPQ $.
+      ( wex wmo wa weu exbid mobid anbi12d df-eu 3bitr4g ) ABDGZBDHZICDGZCDHZIB
+      DJCDJAPRQSABCDEFKABCDEFLMBDNCDNO $.
   $}
 
   ${
-    mobid.1 $e |- F/ x ph $.
-    mobid.2 $e |- ( ph -> ( ps <-> ch ) ) $.
-    $( Formula-building rule for the at-most-one quantifier (deduction form).
-       (Contributed by NM, 8-Mar-1995.) $)
-    mobid $p |- ( ph -> ( E* x ps <-> E* x ch ) ) $=
-      ( wex weu wi wmo exbid eubid imbi12d moeu 3bitr4g ) ABDGZBDHZICDGZCDHZIBD
-      JCDJAPRQSABCDEFKABCDEFLMBDNCDNO $.
+    $d x y $.  $d ph y $.
+    $( An alternate way to express existential uniqueness.  (Contributed by NM,
+       8-Jul-1994.)  Add a disjoint variable condition on ` ph , y ` .
+       (Revised by Wolf Lammen, 29-May-2019.) $)
+    eu3v $p |- ( E! x ph <-> ( E. x ph /\ E. y A. x ( ph -> x = y ) ) ) $=
+      ( weu wex wmo wa weq wi wal df-eu df-mo anbi2i bitri ) ABDABEZABFZGOABCHI
+      BJCEZGABKPQOABCLMN $.
+  $}
+
+  ${
+    $d x y z $.
+    $( There exists a unique set equal to a given set.  Special case of ~ eueqi
+       proved using only predicate calculus.  The proof needs ` y = z ` be free
+       of ` x ` .  This is ensured by having ` x ` and ` y ` be distinct.
+       Alternately, a distinctor ` -. A. x x = y ` could have been used
+       instead.  See ~ eueq and ~ eueqi for classes.  (Contributed by Stefan
+       Allan, 4-Dec-2008.)  (Proof shortened by Wolf Lammen, 8-Sep-2019.) $)
+    euequ $p |- E! x x = y $=
+      ( vz weq weu wb wal wex ax6evr equequ2 alrimiv eximii eu6 mpbir ) ABDZAEO
+      ACDFZAGZCHBCDZQCCBIRPABCAJKLOACMN $.
+  $}
+
+  $( old theorems $)
```

Comment:
I think this comment can be removed.

Reply:
I planned on removing it (and the comment `($ substitution $)`) when the whole section is treated. Maybe two or three more PRs.  In the meantime, it helps me keep track of what has been done. Is that ok with you ?